mkdocs.yml

```diff
@@ -27,6 +27,9 @@ nav:
           - Install nomad-lab: howto/programmatic/pythonlib.md
           - Access processed data: howto/programmatic/archive_query.md
           - Transform data: howto/programmatic/json_transformer.md
+      - Graph-style API:
```

Comment:
Put the section "Graph-style API" under "Programmatic use". This is where it fits currently best.